tests/Acceptance/ApplicationTest.php

```diff
@@ -94,7 +95,7 @@ public function test_environment_deferred_application_execution(): void
         $this->assertStringContainsString(' echo "test"', MockWriter::$content);
         $this->assertStringContainsString('All commands successfully executed!', MockWriter::$content);
         $this->assertStringContainsString('Duration:', MockWriter::$content);
-        self::assertStringEqualsFile(__DIR__ . '/_app/result.txt', 'test');
+        $this->assertStringEqualsFile(__DIR__ . '/_app/result.txt', 'test');
```

Comment:
Previous implementation was correct.
```suggestion
        self::assertStringEqualsFile(__DIR__ . '/_app/result.txt', 'test');
```